eZ/Publish/Core/MVC/Symfony/Routing/Generator/UrlAliasGenerator.php

```diff
@@ -55,11 +55,20 @@ class UrlAliasGenerator extends Generator
      */
     private $configResolver;
 
-    public function __construct( Repository $repository, RouterInterface $defaultRouter, ConfigResolverInterface $configResolver )
+    /**
+     * Array of characters that are potentially unsafe for output for (x)html, json, etc,
+     * and respective url-encoded value
+     *
+     * @var array
+     */
+    protected $unsafeCharMap;
```

Comment:
Why protected? Please always prefer `private` as it's always a good way to keep BC.